### DISCUSSION TITLE
How to sync evolu server with app

### DISCUSSION THREAD

**Discussion by freeseamew** [evoluhq/evolu#469] [Q&A]
I am very interested in the evolu method.

I am asking because there is a part of the document that I do not know how to use.

It is about data synchronization between the evolu client app and the evolu-server.

The document shows how to run the evolu server,

but I do not see any part about how to synchronize the server and the app.

Could you please give me some hints about this part?

**Comment by steida** [comment]
Here https://www.evolu.dev/docs/how-evolu-works, but soon, it will be massively improved.
